pkg/proxy/proxy.go

```diff
@@ -133,7 +141,7 @@ func Deploy(sso *apiv1.SSO, oidcClient *api.Client, cookieSecret string) (*Proxy
 		},
 	}
 
-	deployment := buildName(sso.GetName(), sso.GetNamespace())
+	deployment := buildName(sso.GetName(), "")
```

Comment:
Add the namespace looks not useful as the resource is only available in a given namespace.
Although adding it makes the name be more than 63 chars quite easily then we end up with a ugly truncated string.